### ISSUE TITLE
Linux install script error on Debian

### ISSUE THREAD

**Issue by jase64** [pgadmin-org/pgadmin4#7081]
Hi Community,

I noticed an error occurring (using Debian 11.8) in the install script given in the web page https://www.pgadmin.org/download/pgadmin-4-apt/

The ```lsb_release -cs``` in the line:
```
# Create the repository configuration file:
sudo sh -c 'echo "deb [signed-by=/usr/share/keyrings/packages-pgadmin-org.gpg] https://ftp.postgresql.org/pub/pgadmin/pgadmin4/apt/$(lsb_release -cs) pgadmin4 main" > /etc/apt/sources.list.d/pgadmin4.list && apt update'
```

should rather read ```lsb_release -c -cs```

I hope this may help other newcomers in avoiding troubleshooting the install by themself.

Anyway thx to Community for the very useful PgAdmin4.

**Comment by khushboovashi** [comment]
Hi @jase64, 

Please provide the error details which you got while installing.

**Comment by jase64** [comment]
Hi @khushboovashi,

I unfortunately won't be able to reproduce to error encountered since in the meantime I destroyed the VM where it happened. I may have faced a "buggy" distribution of Debian 11 (release name did show only up using ```lsb_release -c -cs```).

Upon your question, I tried again on Debian 12 (linux dedian 6.1.0-17-amd64) and I everything went fine using ```lsb_release -cs```.

Sorry then for what might have been a false alert.
